src/services/eth.js

```diff
@@ -123,4 +127,40 @@ export default class Ethereum {
       return reason
     }
   }
+
+  // get remote token list
+  async getTokenList (tokenListUrl) {
+    try {
+      axios.get(tokenListUrl)
```

Comment:
try catch construction should be use in case if you using async/await. IN another case error will in order execution flow and you cannot catch it.
You should write `await` before `axios.get` and use async await flow for this operation.